images/runtime/php/8.0/base.buster.Dockerfile

```diff
@@ -19,6 +19,12 @@ RUN echo -e '<FilesMatch "\.(?i:ph([[p]?[0-9]*|tm[l]?))$">\n SetHandler applicat
 # Disable Apache2 server signature
 RUN echo -e 'ServerSignature Off' >> /etc/apache2/apache2.conf
 RUN echo -e 'ServerTokens Prod' >> /etc/apache2/apache2.conf
+RUN { \
```

Comment:
Could you please add a comment in the Dockerfile to explain what this is doing?

Reply:
It was auto generated when I run the `./buildRuntimeImages.sh` or `./buildRuntimeImageBases.sh` script locally. It happened previously and I think we're good to add these to the `conf` file as the default configuration.